tools/datagen/README.md

```diff
@@ -11,39 +11,39 @@ More details on each tool can be found by running `--help`.
 
```

Comment:
Nit: Here and in main.rs: Please add

```
3. `icu4x-key-extract`: Extract `ResourceKey` objects present in a compiled executable.
```

or remove all references to icu4x-keyextract until we actually add that tool